doc/conf.py

```diff
@@ -44,6 +44,10 @@
     'zephyr.application',
 ]
 
+# Only use SVG converter when it is really needed, e.g. LaTeX.
```

Comment:
BTW I started using DOT_IMAGE_FORMAT = svg for Doxygen HTML output everywhere I can and I never looked back. Faster compile times, zoomable output and much, MUCH smaller disk usage. Doxygen hasn't changed the default yet because they don't want to break... IE9?

Just FYI.